docs/semgrep-app/pricing-and-billing.mdx

```diff
@@ -1,53 +1,167 @@
+| **Member limit** | ♾️ unlimited | ♾️  unlimited | ♾️  unlimited |
+| **Project limit** |♾️  unlimited | ♾️  unlimited | ♾️  unlimited |
+
+#### Findings, language support, and rules
+
+| Feature  | Community | Team | Enterprise |
+| -------  | --------- | ------- | -------- | 
+| **Findings retention** | 1 month | 1 year | 1 year+ (custom) |
+| **Community rule registry** | ✔️ yes | ✔️  yes  | ✔️  yes | 
+| **Named rules** | ✔️ yes | ✔️ yes | ✔️  yes |
+| **Private rules** | ❌ no | ✔️ yes | ✔️  yes |
+| **Custom rule messages** | ❌ no | ✔️  yes | ✔️  yes |
+| **Custom language support** | ❌ no | ❌ no | ✔️  yes |
+
+Notes:
+
+* **Named rules** are rules that have been saved through the [Playground](https://semgrep.dev/editor). The Playground is a live editor where you can write your rule patterns and test them on sample code. Named rules created by Community tier members are public and available in the Playground.
+* **Custom rule messages** can be applied to public and private rules.
+
+#### Deployment, integrations, and notifications
+
+| Feature  | Community | Team | Enterprise |
+| -------  | --------- | ------- | -------- | 
+| **Automatic CI/CD** | ✔️ yes | ✔️  yes  | ✔️  yes | 
+| **Slack notifications** | ✔️ yes | ✔️  yes  | ✔️  yes | 
+| **Email notifications** | ✔️ yes | ✔️  yes  | ✔️  yes | 
+| **Webhook integration** | ❌ no | ✔️  yes | ✔️  yes |
+| **Jira integration** | ❌ no | ✔️  yes | ✔️  yes |
+| **VPC deployment** | ❌ no | ❌ no | ✔️  yes |
+
+Notes:
+
+* There is no limit to the number of integrations or notification channels. You can make more than one integration of any type.
+* **Webhook integration** uses a generic webhook to send JSON messages. These messages are triggered when a policy is changed, a new scan runs, or a new finding has emerged.
+* **Jira integration** allows organizations to directly create Jira tickets from a finding.
+
+#### Authentication
+
+| Feature                                  | Community | Team | Enterprise |
+| ------                                    | --------- | ------- | -------- | 
+| **GitHub or GitLab single sign-on (SSO)** | ✔️ yes | ✔️  yes  | ✔️  yes | 
+| **SAML SSO** | ❌ no | ✔️  yes | ✔️  yes |
+| **Role-based access control (RBAC)** | ❌ no | ✔️  yes | ✔️  yes |
+| **Custom authentication features** | ❌ no |  ❌ no | ✔️  yes |
+
+Notes:
+
+* Available **RBAC roles** are `admin` and `user`.
+
+#### Support
+
+| Feature  | Community | Team | Enterprise |
+| -------  | --------- | ------- | -------- | 
+| **Slack support** | Community Slack | Private Slack channel | Private Slack channel |
+| **Semgrep support portal** | ❌ no | ✔️  yes | ✔️  yes |
+| **Customer success manager** | ❌ no | ❌ no | ✔️  yes |
+| **Dedicated and customized onboarding** | ❌ no | ❌ no | ✔️  yes |
+
+Notes:
+
+* Request support and track the status of your tickets through the [Semgrep Support Portal](https://support.semgrep.dev/hc/en-us), which is available 8 hours a day, 5 days a week.
```

Comment:
IMO this reads as if the website is only available 8x5, but it’s in fact the (human) support that is available 8x5. The support portal is available for requesting support and tracking status of tickets all the time.

Reply:
Good catch. I have corrected this.